### ISSUE TITLE
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such table: json_each , when first start prefect

### ISSUE THREAD

**Issue by seabnavin19** [PrefectHQ/prefect#7916]
### First check

- [X] I added a descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Prefect documentation for this issue.
- [X] I checked that this issue is related to Prefect and not one of its dependencies.

### Bug summary

I just install prefect in my virtual environment and start a basic function in prefect, but when I first start flow it show me this erorr
``` bash 
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such table: json_each
[SQL: INSERT INTO flow_run_notification_queue (flow_run_notification_policy_id, flow_run_state_id) SELECT flow_run_notification_policy.id, CAST(:param_1 AS CHAR(36)) AS anon_1
FROM flow_run_notification_policy
WHERE flow_run_notification_policy.is_active IS 1 AND (flow_run_notification_policy.state_names = :state_names_1 OR EXISTS (SELECT 1
FROM json_each(flow_run_notification_policy.state_names) AS json_each
WHERE json_each.value IN (:json_each_values_1_1))) AND (flow_run_notification_policy.tags = :tags_1 OR EXISTS (SELECT 1
FROM json_each(flow_run_notification_policy.tags) AS json_each
WHERE json_each.value IN (SELECT 1 FROM (SELECT 1) WHERE 1!=1)))]
[parameters: {'param_1': 'ba74eef6-b05b-4f3c-b45d-6ae4970e0b69', 'state_names_1': '[]', 'tags_1': '[]', 'json_each_values_1_1': 'Scheduled'}]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
````

### Reproduction

```python3
from prefect import flow
@flow()
def hi():
    print("hi")

if __name__ =="__main__":
    hi()
```

### Error

```python3
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such table: json_each
[SQL: INSERT INTO flow_run_notification_queue (flow_run_notification_policy_id, flow_run_state_id) SELECT flow_run_notification_policy.id, CAST(:param_1 AS CHAR(36)) AS anon_1
FROM flow_run_notification_policy
WHERE flow_run_notification_policy.is_active IS 1 AND (flow_run_notification_policy.state_names = :state_names_1 OR EXISTS (SELECT 1
FROM json_each(flow_run_notification_policy.state_names) AS json_each
WHERE json_each.value IN (:json_each_values_1_1))) AND (flow_run_notification_policy.tags = :tags_1 OR EXISTS (SELECT 1
FROM json_each(flow_run_notification_policy.tags) AS json_each
WHERE json_each.value IN (SELECT 1 FROM (SELECT 1) WHERE 1!=1)))]
[parameters: {'param_1': 'ba74eef6-b05b-4f3c-b45d-6ae4970e0b69', 'state_names_1': '[]', 'tags_1': '[]', 'json_each_values_1_1': 'Scheduled'}]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
```

### Versions

```Text
Name: prefect
Version: 2.6.1
Summary: Workflow orchestration and management.
Home-page: https://github.com/PrefectHQ/prefect/tree/orion
Author: Prefect Technologies, Inc.
Author-email: help@prefect.io
License: None
Location: d:\project\jira-support\venv_1\lib\site-packages
Requires: aiosqlite, pytz, sqlalchemy, python-slugify, rich, anyio, apprise, coolname, alembic, griffe, aiohttp, pydantic, docker, readchar, typing-extensions, importlib-metadata, croniter, fastapi, jsonpatch, packaging, pyyaml, httpx, uvicorn, fsspec, pendulum, cryptography, orjson, aiofiles, asgi-lifespan, asyncpg, kubernetes, pathspec, click, typer, cloudpickle, toml
```

**Comment by madkinsz** [comment]
@seabnavin19 Can you share your SQLite versions? (these are displayed by `prefect version` as well)

Generally this means your SQLite is too old and does not support the `json_each` operation.

**Comment by madkinsz** [comment]
Note this may be a duplicate of https://github.com/PrefectHQ/prefect/issues/5970
